assignment02/script.js

```diff
@@ -1,72 +0,0 @@
-
- var css_link = $("<link>", {
-    rel: "stylesheet",
-    type: "text/css",
-    href: "oz.css"
-});
-
-css_link.appendTo('head');
-
-$(document).ready(function(){
-
-
-   //1------------------------------------------------------------ #1
-    var women=prompt("Enter the name of the lady", " ");
-    var lady= ['#lady','#lady1','#lady2'];
```

Comment:
you weren't required to use an array and loop to solve this assignment - but you've got it working well.